### ISSUE TITLE
Simple single notification

### ISSUE THREAD

**Issue by blu3beri** [arnnis/react-native-toast-notifications#109]
Hiya,

I was wondering what the best method is for only allowing a single notification on the screen.

I have called `toast.hideAll()` before calling `toast.show(...)` but it results in some animation bugs.

Any idea?

cheers

edit:

This seems to work as a patch.

```tsx
  const run = async () => {
    toast.hideAll()
    await new Promise<void>((resolve) => setTimeout(() => resolve(), 500))
    toast.show(text, { type: variant })
  }
  void run()
```

**Comment by devthejo** [comment]
I've fixed it by #114

**Comment by devthejo** [comment]
@blu3beri you don't merge the pull request ?

**Comment by blu3beri** [comment]
I am not a contributor of this repository. It is probably up to @arnnis

**Comment by devthejo** [comment]
Ho, sorry for the confusion, I thought that because of closing the issue, this must be kept open until merged IMHO.

**Comment by arnnis** [comment]
This is now fixed in version [3.3.0](https://github.com/arnnis/react-native-toast-notifications/releases/tag/v3.3.0).
